code/modules/power/singularity/collector.dm

```diff
@@ -37,6 +37,8 @@
 			eject()
 		else
 			loaded_tank.air_contents.gases[/datum/gas/plasma][MOLES] -= 0.001*drainratio
+			ASSERT_GAS(/datum/gas/tritium,loaded_tank.air_contents)
+			loaded_tank.air_contents.gases[/datum/gas/tritium][MOLES] += 0.001*drainratio

```

Comment:
oh god it perfectly converts all the plasma?

Also this is going to make collector spam a thing again, don't worry though I'll fix that in the next major balance pass.